main.nf

```diff
@@ -1,6 +1,9 @@
 process GENERATE_FAKE_FASTQ {
     container 'community.wave.seqera.io/library/numpy:2.1.1--3063fc3d721f2cdf'
 
+    cpus 2
+    memory { 4.GB * task.attempt }
```

Comment:
Still not sure about this as a default 😅 It can mask performance issues, especially if we are ignoring failed tasks in the reporting.

Reply:
For me is the same problem that we have when AWS Batch runs it on a different EC2 VM, it's also masking performance issues.

Reply:
Fair point. Let's maybe keep an eye out for failed tasks and see how often this happens. Ideally, we come back here and update the default resources if required so we don't have to scale up.

Reply:
I've tested these values, and I don't expect them to retry (at least with Fusion).